src/Http/Organizer/GetPermissionsForCurrentUserRequestHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Organizer;
+
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Http\Response\PrivateJsonResponse;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use CultuurNet\UDB3\Security\Permission\PermissionVoter;
+use CultuurNet\UDB3\Security\Permission\UserPermissionChecker;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+final class GetPermissionsForCurrentUserRequestHandler implements RequestHandlerInterface
```

Comment:
I think there is no need to duplicate this code, you can add a new instance of `GetPermissionsForCurrentUserRequestHandler` to the container with another name and set the organizer permission.

Reply:
But that class is in the `Offer` namespace. Wouldn't it be better to keep them separated, even if that means duplicating a little bit of code? We also do that for other features that exist both on organizers and offers like for example labels.